ui/job-view/App.jsx

```diff
@@ -388,6 +386,8 @@ class App extends React.Component {
             <Notifications />
             {showShortCuts && (
               <div
+                role="button"
+                tabIndex={0}
```

Comment:
As much as possible, I think we should be turning elements that behave like buttons into semantic button elements. Marco mentioned use of semantic elementsduring the zoom meeting a few months ago and it's also mentioned in the eslint rule description :)

Reply:
Specifically for this component, I think it would be better to leave it as a `div`. This encapsulates the whole  view below:

![01](https://user-images.githubusercontent.com/3901809/74967033-fa065280-53f6-11ea-904f-a983b4d4a333.png)

All of the other interactive `span` or `div` were transformed into reactstrap `Button`.

Reply:
Yeah, I would agree this one is a special case.  Not sure if we could handle this in a different way (perhaps there's a Component we could use here instead that handles clicks to close the modal?)  But This seems fine for now.

Reply:
Ok, so it looks like this div is acting like a modal, so I think giving it a role of button might be confusing. It can be replaced with a reactstrap modal.